wagtail/contrib/simple_translation/tests/test_wagtail_hooks.py

```diff
@@ -231,6 +232,16 @@ def test_translation_count_in_context(self):
         # Create blog_post copies for translation
         self.fr_blog_post = self.en_blog_post.copy_for_translation(self.fr_locale)
 
+        # Create an alias page to test the `translations_to_move_count`
+        # in the template context
+        CreatePageAliasAction(
+            self.fr_blog_post,
```

Comment:
The goal of the test is to check it ony counts the translations of the source page, not its alias.

This is incorrect in that you're creating an alias of the translated page, rather than the source page (which is the one that gets moved just below)

The best thing to do here is:

Create another tree (e.g. german), then create an alias of the source page (`self.en_blog_post`) in the german tree. You can use `copy_for_translation()` with `alias=True`

The test on `translations_to_move_count == 1` should still be valid